pkg/disk/partition_table.go

```diff
@@ -867,6 +871,9 @@ func (pt *PartitionTable) features() partitionTableFeatures {
 			ptFeatures.Swap = true
 		case *LUKSContainer:
 			ptFeatures.LUKS = true
+		case *PartitionTable, *Partition:
```

Comment:
(super nitpick) - I personally like a small comment like `// nothing`, `// pass`, `// nothing to do` or similar for empty cases do clarify that this is intentionally left blank and for those faimilar with C it remind them that this is not an auto-fallthrough

Reply:
Done!